tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1324,3 +1324,40 @@ def test_deleted_idm_user_should_not_be_able_to_login(auth_source_ipa, ldap_tear
         with raises(NavigationTriesExceeded) as error:
             ldapsession.user.search('')
         assert error.typename == "NavigationTriesExceeded"
+
+
+@tier2
+def test_onthefly_functionality(session, ipa_data, ldap_tear_down):
+    """IDM user will not be created automatically in Satellite if onthefly is
+    disabled
```

Comment:
In test, you check that user can't log in. Would it make sense to explicitly check if user is not created?

Reply:
The user will only be created in satellite after the user has been authenticated from LDAP(here IDM). So being said that, the user creation depends on the successful login.
Even if we `'account.onthefly_register': True`, until the user logs in, the account will not be created.